src/configure/configure.ts

```diff
@@ -173,7 +173,11 @@ class Orchestrator {
                 this.inputs.azureSession = getAzureSession();
             }
             const repoAnalysisResult = await this.getRepositoryAnalysis();
-            await this.getSelectedPipeline(repoAnalysisResult);
+            if (resourceNode && resourceNode.type === TargetResourceType.AKS) {
```

Comment:
Isn't it the same thing?
Anyway we'll have to put up a condition on the resourceNode type itself to put the required values for deployTarget and language, so instead of ``` context['isResourceSelected] && resourceNode.type === TargetResourceType.AKS ```, the existing one looke better?